apps/allocations/app/components/App/App.js

```diff
@@ -90,15 +99,15 @@ const App = () => {
     })
   }
 
-  const onNewAllocation = (address, description, id, balance) => {
+  const onNewAllocation = (id, description, balance, token) => {
+    console.log('budgetId: ', id)
     setPanel({
       content: NewAllocation,
       data: {
         heading: 'New Allocation',
         subHeading: description,
-        address,
         balance,
-        balances,
+        balances: [token],
```

Comment:
The confusion of the word "budget" and "token" here seems like something we should talk about at some point. This seems like it might indicate that it's about time to reshape our state tree and rename some of its attributes. But we can have that conversation later.